scanomatic/image_analysis/image_basics.py

```diff
@@ -60,10 +61,11 @@ def Quick_Scale_To_im(
 ) -> np.ndarray:
     if im is None:
         try:
-            im = load_image_to_numpy(path, dtype=np.uint8)
+            im = load_image_to_numpy(str(path), dtype=np.uint8)
```

Comment:
I think it better to before the try raise an `LoadImageError` if `path is None and im is None`

Reply:
Fixed